infinispan-remote/src/test/java/org/hibernate/ogm/datastore/infinispanremote/test/storedprocedures/ResultSetProcedure.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+
+package org.hibernate.ogm.datastore.infinispanremote.test.storedprocedures;
+
+import java.util.Collections;
+import java.util.List;
+
+import org.hibernate.ogm.backendtck.storedprocedures.Car;
+
+import org.infinispan.tasks.ServerTask;
+import org.infinispan.tasks.TaskContext;
+
+/**
+ * @author The Viet Nguyen
+ */
+public class ResultSetProcedure implements ServerTask<List<Car>> {
+
+	private int id;
+	private String title;
+
+	@Override
+	public void setTaskContext(TaskContext taskContext) {
+		id = taskContext.getParameters()
```

Comment:
Maybe could be nice having a similar behaviour of the corresponding js file: __resultSetProcedure.js__.
In term of write and read data from a real cache.

Reply:
sure, added similar operations